modules/checklist/steps/set-fonts-and-colors.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Elementor\Modules\Checklist\Steps;
+
+use Elementor\Core\DocumentTypes\Page;
+use Elementor\Plugin;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Set_Fonts_And_Colors extends Step_Base {
+	const STEP_ID = 'set_fonts_and_colors';
+
+	public function get_id() : string {
+		return self::STEP_ID;
+	}
+
+	public function is_absolute_completed() : bool {
+		$settings = $this->kit_adapter->get_kit_settings();
+		$custom_color = $settings['custom_colors'] ?? '';
+		$custom_fonts = $settings['custom_typography'] ?? '';
+
+		return ! empty( $custom_color ) && ! empty( $custom_fonts );
+	}
+
+	public function get_title() : string {
+		return __( 'Set up your Global Fonts & Colors', 'elementor' );
+	}
+
+	public function get_description() : string {
+		return esc_html__( 'Global colors and fonts ensure a cohesive look across your site. Start by defining one color and one font.', 'elementor' );
+	}
+
+	public function get_cta_text() : string {
+		return esc_html__( 'Go to Site Identity', 'elementor' );
+	}
+
+	public function get_cta_url() : string {
+		$link = Page::get_site_settings_url_config( true )['url'];
```

Comment:
Why bool? The function accepts an id parameter
`get_site_settings_url_config( $active_tab_id = null )`